superset/connectors/sqla/models.py

```diff
@@ -688,7 +687,8 @@ def select_star(self) -> Optional[str]:
 
     @property
     def health_check_message(self) -> Optional[str]:
-        return self.extra_dict.get("health_check", {}).get("message")
+        check = config["DATASET_HEALTH_CHECK"]
+        return check(self) if check else None
```

Comment:
what's more pythonic? this or:
```suggestion
        return (config["DATASET_HEALTH_CHECK"] or lambda: None)(self)
```

is this even valid python? 🤷